frontend/app/src/components/StreamlitDialog/StreamlitDialog.test.tsx

```diff
@@ -86,14 +97,16 @@ describe("aboutDialog", () => {
       </Fragment>
     )
 
-    expect(wrapper.find(Modal).text()).toContain("Streamlit v42.42.42")
+    const versionRegex = /Streamlit v\s*42\.42\.42/
+    const versionText = screen.getByText(versionRegex)
+    expect(versionText).toBeDefined()
```

Comment:
Nit: does checking for "Streamlit v42.42.42" like before not work properly?
If so, could you add explanatory comment on why regex needed instead of simple string

Reply:
Also think we should keep the Modal render check as part of this test

Reply:
done